code/src/pages/dashboard/dashboard-orders/DashboardOrdersPage.test.tsx

```diff
@@ -182,6 +182,23 @@ describe("DashboardOrdersPage", () => {
     expect(mockApplyFilters).toHaveBeenCalled();
   });
 
+  test("resets search param if input is submited without value", async () => {
+    renderAndMock();
+
+    const searchInput = screen.getByPlaceholderText(
+      /dashboardTabs.orders.search/
+    );
+
+    await typeIntoInput(searchInput, "user");
+    await typeIntoInput(searchInput, "");
```

Comment:
Why do we need to type one value and then another?

Reply:
This test checks if the function is called, that function is called when user clicks search button with empty input. For some reason, it doesn't work other way. I tried to omit typeIntoInput, because input should be empty by default, and also tried yo use only `await typeIntoInput(searchInput, "");`, without previously setting `user.` Both ways doesn't work for some reason